lib/grover.rb

```diff
@@ -65,6 +65,12 @@ def self.convert_function(convert_action)
               await page.authenticate({ username, password });
             }
 
+            // Setting cookies
+            const cookies = options.cookies; delete options.cookies
+            if (cookies != undefined && Array.isArray(cookies)) {
```

Comment:
not sure you need the `undefined` guard. `Array.isArray` should return `false` for `undefined`